crates/pipeline-types/src/server.rs

```diff
@@ -0,0 +1,32 @@
+use serde::Deserialize;
```

Comment:
perhaps these declarations should be in http.rs and we don't need the server.rs module.